File: leetcode1791/solution.cpp
class Solution {
public:
    int findCenter(vector<vector<int>>& edges) {
        /// Time: O(1) | Space: O(1)
        if (edges[0][0] == edges[1][0] || edges[0][0] == edges[1][1] ) {
            return edges[0][0];
        } else {
            return edges[0][1];
        }
    }
};

File: leetcode1791/solution.rs
impl Solution {
    pub fn find_center(edges: Vec<Vec<i32>>) -> i32 {
        /// Time: O(1) | Space: O(1)
        return if edges[0][0] == edges[1][0] || edges[0][0] == edges[1][1] {
            edges[0][0]
        } else {
            edges[0][1]
        };
    }
}